app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,13 @@
+<% provide(:title, t(".title")) %>
+
+
+    <h1><%= t(".h1_title") %></h1>
+    <p>
+      The <%= link_to("Ruby on Rails Tutorial", "http://www.railstutorial.org/") %> is a
+      <%= link_to("book", "http://www.railstutorial.org/book") %> and
+      <%= link_to("screencast series", "http://screencasts.railstutorial.org/") %>
```

Comment:
`"Ruby on Rails Tutorial"`
sefl chekc lại, còn lỗi chưa dùng I18n

xem lại indent file này

sefl check toàn bộ file bên dưới